sc-kpm/sc-agents-common/utils/AgentUtils.cpp

```diff
@@ -143,7 +118,7 @@ ScAddr AgentUtils::getActionResultIfExists(
     const ScAddr & actionNode,
     const sc_uint32 & waitTime)
 {
-  SC_CHECK_PARAM(actionNode, ("Invalid action node address"));
+  SC_CHECK_PARAM(actionNode, "Invalid action node address passed to `getActionResultIfExists`");
```

Comment:
It is not only get action result, it also applies action. We can get action result after it applies and before. Name this method to show that agent is called inside it